demos/node-classification/gcn/gcn-cora-node-classification-example.ipynb

```diff
@@ -4,14 +4,36 @@
    "cell_type": "markdown",
```

Comment:
<p>Why the increase from 40 to 200 epochs here - did you aim to be testing the 'patience' setting which wasn't relevant before?</p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/stellargraph/stellargraph/pull/1125/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='38'/>

Reply:
Not so much testing, as better practice to train for a while and have a relatively high patience. https://github.com/stellargraph/stellargraph/pull/1125#discussion_r396158701